signer/src/bitcoin/packaging.rs

```diff
@@ -603,6 +596,28 @@ mod tests {
             Self { votes, ..Default::default() }
         }
 
+        /// Create a new request item with random votes against.
+        fn with_rng(rng: &mut impl Rng) -> Self {
+            let mut votes = [false; 5];
+            for vote in &mut votes {
+                *vote = rng.gen_bool(0.5);
+            }
+            let needs_signature = rng.gen_bool(0.5);
+            let vsize = rng.gen_range(1..=10000);
+            let withdrawal_id = if needs_signature && rng.gen_bool(0.5) {
+                Some(rng.gen_range(1..=100))
```

Comment:
Having the withdrawal request be random is okay, but they are supposed to be unique (not globally, but by the time they get to this modules code path they should be or we have problems). In the `utxo` module we use a static `AtomicU64` these IDs. This code shouldn't be a problem at all though.

Reply:
[6272373](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/6272373dcd131105f9aa9059788570e14dfcc5a8)